### DISCUSSION TITLE
Persistent server credentials in Docker?

### DISCUSSION THREAD

**Discussion by snppls** [jeffvli/feishin#452] [General]
What folder or file should I map in Docker to make Feishin remember my server credentials? Every time I restart the app I am asked to fill out the details again.

Cheers!

**Comment by kgarner7** [comment]
What do you mean by restart app? Restart the browser? Delete your cookies? What server?
In both cases, the credentials are saved in local storage (`store_authentication`), but for Navidrome the credentials are temporary (expire in 24 hours by default), so if you don't use the webapp for that time period the credentials will expire. Unlike the desktop application, there is no way to save your Navidrome password (what would be required to reauthenticate).
